woocommerce/woocommerce-android
# Issue/5961 Add payment method, country, currency for  IPP events

Closes: #5961


### Description

The PR adds:
* Currency
* Payment method type
* Card reader model

To all (if available) IPP events

### Testing instructions

Try to use the app and check what is tracked via logcat:

```
Tracked: card_present_collect_payment_success, Properties: {"plugin_slug":"woocommerce-payments","country":"US","currency":"USD","payment_method_type":"card_interac","card_reader_model":"CHIPPER_2X","blog_id":192152755,"is_wpcom_store":true,"is_debug":true}
```
